dashboard/config/foorm/forms/surveys/pd/summer_workshop_post_survey.0.json

```diff
@@ -459,6 +897,25 @@
      "allowAddPanel": false,
      "allowRemovePanel": false
     },
+    {
+     "type": "html",
+     "name": "clickTo119",
+     "html": "<p><span style=\"font-size: 14pt; font-family: verdana, geneva, sans-serif;\"><strong>Overall Workshop Feedback</strong></span></p>\n<p>We’d like to hear your feedback about your experience this week overall, <Strong>especially how this professional development could better fit your learning needs, or how it could be improved</strong>.\n"
```

Comment:
sometimes for the section title you use < h2 >, other times < p >, is that intentional?

Reply:
fixed.  Hopefully consistent use of <h2> now.  Tried to remove instances of hard-coded styles.